src/store/reducers/formSettingReducer.js

```diff
@@ -0,0 +1,25 @@
+import {formState} from "../../utils/constants.js";
+const initialState = formState.init;
+
+export function formSettingReducer(state = initialState, action) {
+  const {payLoad, type} = action;
+  switch (type) {
+    case "SET_SETTINGS":
```

Comment:
`"SET_SETTINGS"` and other action types should be defined only once as `constants` and used all over the code not to have a change to make a mistake in the `string`


```js

export const SET_SETTINGS = "SET_SETTINGS"


```